.github/workflows/examples.yml

```diff
@@ -30,14 +31,9 @@ jobs:
         key: ${{ runner.os }}-go-pkg-mod-${{ hashFiles('go.sum') }}
     - uses: actions/setup-go@v2
       with:
-        go-version: ${{ matrix.go_version }}
+        go-version: ${{ env.GO_VERSION }}
     - name: go build
-      run: |
-        GOOS=$(go env GOOS)
-        GOARCH=$(go env GOARCH)
-        # Substitute as latest release
-        VERSION=$(git describe --abbrev=0 --match='v*.*.*' --tags || echo -n "v99.99.99")
-        go build -o ${TF_PLUGIN_CACHE_DIR}/${GOOS}_${GOARCH}/terraform-provider-aws_${VERSION}_x4 .
+      run: go build -o terraform-plugin-dir/terraform-provider-aws_v99.99.99_x5 .
```

Comment:
How is `terraform-plugin-dir` being picked up? 😄  This is passing right now because its just downloading the latest released provider version (`- Downloading plugin for provider "aws" (hashicorp/aws) 3.9.0...` in the [logs](https://github.com/terraform-providers/terraform-provider-aws/runs/1222392266?check_suite_focus=true)), which works fine most of the time, but wouldn't work for new examples with unreleased resources.

Reply:
Yeah, I noticed that when I was testing. That was also the behaviour before this change, so I figured I'd leave it and fix later. Part of the problem is that at least one of the examples uses additional providers, causing problems with `terraform init -plugin-dir=terraform-plugin-dir`.

Reply:
Actually, it was easy…